roles/nodepool/files/etc/nodepool/elements/bonnyci-nodepool/install.d/90-test-logs-dir

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# dib-lint: disable=setu setpipefail
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -e
+
+# Create a log directory for tests to stuff things to be archived by publisher.
+# This path is also set in the job definition and exported to in-repo test
+# scripts, at ./roles/zuul/files/jobs/bonnyci.yaml.
+mkdir -p /opt/bonnyci/logs
+chown bonnyci:bonnyci /opt/bonnyci/logs
```

Comment:
We'll be referencing this path on the zuul config side, in the job's publisher. I'd rather it be a static full path that is indepenent of the user's home dir, which could change for some reason in the future.